app/src/main/java/de/westnordost/streetcomplete/quests/bike_rental_type/AddBikeRentalType.kt

```diff
@@ -0,0 +1,37 @@
+package de.westnordost.streetcomplete.quests.bike_rental_type
+
+import de.westnordost.streetcomplete.R
+import de.westnordost.streetcomplete.data.osm.mapdata.Element
+import de.westnordost.streetcomplete.data.osm.mapdata.MapDataWithGeometry
+import de.westnordost.streetcomplete.data.osm.mapdata.filter
+import de.westnordost.streetcomplete.data.osm.osmquests.OsmFilterQuestType
+import de.westnordost.streetcomplete.data.osm.osmquests.Tags
+import de.westnordost.streetcomplete.data.user.achievements.QuestTypeAchievement.BICYCLIST
+
+class AddBikeRentalType : OsmFilterQuestType<BikeRentalType>() {
+
+    override val elementFilter = """
+        nodes, ways with
+          amenity = bicycle_rental
+          and access !~ private|no
+          and (!bicycle_rental or bicycle_rental = yes)
+          and !shop
+    """
+    override val changesetComment = "Add bicycle rental type"
+    override val wikiLink = "Key:bicycle_rental"
+    override val icon = R.drawable.ic_quest_bicycle_rental
+    override val isDeleteElementEnabled = true
+    override val questTypeAchievements = listOf(BICYCLIST)
+
+    override fun getTitle(tags: Map<String, String>) = R.string.quest_bicycle_rental_type_title
+
+    override fun getHighlightedElements(element: Element, getMapData: () -> MapDataWithGeometry) =
+        getMapData().filter("nodes, ways with amenity ~ bicycle_parking|bicycle_rental")
+
+    override fun createForm() = AddBikeRentalTypeForm()
+
+    override fun applyAnswerTo(answer: BikeRentalType, tags: Tags, timestampEdited: Long) {
+        tags["bicycle_rental"] = answer.osmValue
+        if (answer == BikeRentalType.SHOP) tags["shop"] = "rental"
```

Comment:
`shop=bicycle` `service:bicycle:rental=yes` seems preferred and not requiring confirmation

Reply:
From https://wiki.openstreetmap.org/wiki/Tag:amenity%253Dbicycle_rental:

> If the object to map is a bicycle shop which also happens to rent out bicycles, use `shop=bicycle` with `service:bicycle:rental=yes`.

> `bicycle_rental=shop` and `shop=rental`: A staffed, indoor shop whose **main business** is bike rental, e.g. for tourists in a touristical region.

That's why I added the confirmation prompt.

Reply:
maybe there should be options for both?

Reply:
I thought about that first, but it's hard to find clear concise descriptions and meaningful but distinguishable images for both types.

Reply:
`Bicycle shop offering rental` vs `Human-operated bicycle rental`?

Maybe something with less prominent pile of bicycles as a photo? For example https://commons.wikimedia.org/wiki/File:Shkoder_088.JPG https://commons.wikimedia.org/wiki/File:Eizenga_Tweewielers,_Muntendam_(2017).jpg https://commons.wikimedia.org/wiki/File:Ferweda_Service_Center,_Groningen_(2020)_02.jpg https://commons.wikimedia.org/wiki/File:Rijwielspeciaalzaak_E.J._Noordhoff,_Delfzijl_(2019)_01.jpg or something else from https://commons.wikimedia.org/wiki/Category:Bicycle_shops_by_country

Reply:
I'm not convinced by the images you suggested, and also couldn't find any better ones that were easily distinguishable from the bike rental shop image.

> with less prominent pile of bicycles

That also makes it less recognizable as a bicycle shop, as most of them prominently display what they sell (or rent, respectively).

I like your descriptions though. "Human-operated bicycle rental" is better than "Human-operated shop" even if we don't add that other answer, so I changed that already in [`663843c`](https://github.com/streetcomplete/StreetComplete/pull/4010/commits/663843cee71b68913aab83e46168157311078b7f).